### ISSUE TITLE
Batch Kapow remote feed

### ISSUE THREAD

**Issue by CesarGallego** [BBVA/kapow#14]
As Kapow admin I want to replicate a running kapow from a newly fresh kapow, even if i have no original script files.

**Comment by CesarGallego** [comment]
I realize that list must print the original script too

**Comment by nilp0inter** [comment]
The command `kapow route list` which prints out the full list of current routes in JSON format has been implemented in 71bca13f.

The expected behavior can be achieved with a little scripting:

```bash
kapow route list | jq -r '.[]|"curl -XPUT -d\(.|@text) http://localhost:8081/routes"' | sh
```
